lib/src/layout/sidebar/sidebar_item.dart

```diff
@@ -1,6 +1,7 @@
 import 'package:flutter/foundation.dart';
-import 'package:macos_ui/macos_ui.dart';
-import 'package:macos_ui/src/library.dart';
+
+import '../../../macos_ui.dart';
+import '../../library.dart';
```

Comment:
This is just noise, presumably frome something in your IDE setup automatically converting package to relative imports. Please revert this. In VSCode it should be possible to set per-project settings to disable whatever is making this change just for this repo.

Reply:
Apologies. I have to turn this stuff off 🤦